style.css

```diff
@@ -0,0 +1,72 @@
+html{
+    background-color: #ffdd99;
+}
+
+.header{
+    width: 100%;
+    color: #ffdd99;
+    background-color: #336600;
+    text-align: center;
+}
+.header h1{
+    padding: 10px;
+    font-size: 3.5em;
+}
+
+
+.nav ul{
+    background-color: #336600;
+    text-align: center;
+    margin: 0px;
+    padding: 10px;
+    overflow: hidden;
+}
+
+.nav ul li{
+    display: inline;
+    text-align: center;
+    font-size: 2em;
+    padding: 40px;
+    color: #ffdd99;
+}
+
+.ext{
+    display: inline;
+    text-align: center;
+    font-size: 2em;
+    padding: 40px;
+    color: #ffdd99;
+}
+
+
+.nav ul li:hover{
+    background-color: #001a13;
+}
+
+.ext:hover{
+    background-color: #336600;
+}
+
+.nav ul li a{
+    color: #ffdd99;
+    text-decoration: none;
+}
+
+.main{
+    text-align: center;
+    font-size: 2.6em;
+}
+
+.footer{
+    background-color: #336600;
+    padding: 10px;
+}
+
+.footer p{
+    margin: 50px;
+    padding: 50px;
+    text-align: right;
+    display: inline;
+    margin: 20px;
```

Comment:
This margin will overwrite the margin of 50px set above. Remove the one above if it's not needed.